network/core-node/README.mdx

```diff
@@ -15,15 +15,30 @@ If you’re serious about building on Stellar, have a production-level product o
 
 If you’re going the DIY route, this section of the docs is for you. It explains the technical and operational aspects of installing, configuring, and maintaining a Stellar Core node, and should help you figure out the best way to set up your Stellar integration.
 
-The basic flow, which you can navigate through using the menu on the left, goes like this:
+## Node Setup Process
 
-- Choose which type of node you want to run
-- Prepare Your Environment
-- Install Stellar Core
-- Configure Stellar Core
-- Join the network
-- Monitor and maintain your node
-- Join the validators channels to stay on top of critical upgrades and network votes
+The basic flow, which you can navigate through using the "Admin Guide" on the left, goes (roughly) like this:
+
+### Initial Setup
+
+1. Use the information on this _Introduction_ page to determine which type of node you want to run.
```

Comment:
How about providing a link to the `#types-of-nodes` section on this page below to make it easier?